### DISCUSSION TITLE
Conhecimento para favela

### DISCUSSION THREAD

**Discussion by lucasferreiralimax** [frontendbr/forum#1371] [Discussão]
Olá!

O que vcs acham que seria possivel para fomentar o conhecimento dentro das comunidades perifericas do nosso pais para resgatar nossa juventude e direcionar para um futuro melhor?

Hoje em dia a internet possibilita muita coisa, mas o mundo real esta bem distante disso.
Seria interresante criar meetups e eventos dentro das favelas para compartilhar o conhecimento de forma simples e fazer nosso povo crescer cada vez mais e evoluir nosso pais.

Eu sonho em ver o conhecimento salvando vidas e mudando familias, a gente pode melhorar muita coisa basta organizar e acreditar que tudo é possivel.

O que vcs da comunidade front acham sobre projetos dentro das periferias?

**Comment by fernandoRj** [comment]
bom eu morei em favela por anos e posso afirmar que o que faz um jovem entrar para o trafico nao e falta de oportunidade e mts vezes falta de uma base familiar, falta de uma familia para instruir e educar o jovem o que e certo e errado, pode por a melhor sala de tecnologia que o mlk nao vai se interessar em estudar.

E sim existem mts pessoas na favela que podem se interessar por tecnologia, existem mts pessoas honestas que tem uma boa base familiar que sabe educar seus filhos, impor limites, ensiar o que e certo e errado, talvez pra esse tipo de jovem ira ajudar mt

**Comment by Roneycacau** [comment]
Tem a galere do Quebradev se organizando pra isso.

**Comment by israelcena** [comment]
O problema não é a favela, nem a falta de oportunidades, é a facilidade que a vida do crime oferece que acaba atraindo os jovens daquele lugar.
Tenho muitos colegas nos quais eu ofereço oportunidades e os mesmos procuram sempre optar pela "vida mais fácil"

**Comment by dantas-dev** [comment]
Eu vim de comunidade, uma das mais perigosas do RJ. Acredito que seria ótimo. O conhecimento mudou minha vida.

**Comment by didiraja** [comment]
Eu ja me disponibilizei para eventos do tipo. Ensinar HTML básico e explicar como funciona o mercado de Front-End. No caso, o gargalo foi arrumar computador pra galera, seja pras aulas seja em casa.

**Comment by faelplg** [comment]
Grande @lucasferreiralimax 
Espero ter respondido em tempo de que possa visualizar.
Sou de Belo Horizonte, já co-fundei uma empresa de desenvolvimento de softwares e hoje tenho uma empresa que desenvolve tecnologia web para rastreio, coleta e reciclagem de resíduos. 

Por trabalhar na área de sustentabilidade, acabei criando parcerias e uma rede de contatos com as principais entidades que possuem projetos dentro das comunidades. Tem uma que julgo mais interessante para você pesquisar sobre. É o projeto [FA.VELA](https://www.facebook.com/maisfavela/). Eles não são bem uma ONG (possuem fins lucrativos). Ajudam na aceleração de empreendedores de dentro da comunidade e que atuem na própria comunidade. Se você pegar o complexo do morro do papagaio e do aglomerado da Serra, ambos em Belo Horizonte, a população chega a 40 mil pessoas.

Eu quis te passar essas informações, pois esses tipos de empreendimentos precisam MUITO de um apoio tecnológico e eles **têm pouquíssimo acesso a isso**. Outro ponto MUITÍSSIMO IMPORTANTE. **Você não entra na favela sem representatividade**. Para se conectar às comunidades, você precisa se conectar aos porta-vozes e lideranças culturais, principalmente. **Resumindo**, é muito melhor que você faça uma parceria com esses tipos de entidades como o FA.VELA do que tentar uma implementação direta.

Se estiver interessado realmente em criar uma forma de ajudar, dá o grito que me junto a você para ajudar nessas conexões.

Grande abraço.
